### PR TITLE
:sparkles:  A simple helper script to update the version number

### PR DESCRIPTION
**Pull request by ezrasilvera**
## Related issue(s)

Fixes #

### CODE REVIEW

**Review by pdettori — COMMENTED** [review]
/lgtm

File: scripts/update-release.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Input version should be X.Y.Z
```
Comment (MikeSpreitzer) [inline scripts/update-release.sh:3]: No support for X.Y.Z-rc1?

File: scripts/update-release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    exit 1
+fi
+
+# Replace version string using sed
+
+sed -i "/- --version/{n;s/- \"[0-9]\+\.[0-9]\+\.[0-9]\+\"/- \"$input_version\"/}" config/postcreate-hooks/kubestellar.yaml
+
+sed -i "s/export KUBESTELLAR_VERSION=[0-9]\+\.[0-9]\+\.[0-9]\+/export KUBESTELLAR_VERSION=$input_version/" docs/content/direct/examples.md
+
+sed -i "/The latest release is/s/[0-9]\+\.[0-9]\+\.[0-9]\+/$input_version/g" docs/content/direct/README.md
```
Comment (MikeSpreitzer) [inline scripts/update-release.sh:17]: The wording being edited here is very interim. The plan is to focus this doc on saying what is the latest stable release. But we can not do that until we have one.

**Comment by MikeSpreitzer** [comment]
The good news is that this is not a doomsday machine, so keeping it a secret is not so bad.

File: scripts/update-release.sh (hunk 1 above)
Comment (ezrasilvera) [reply scripts/update-release.sh:3]: Good point.

Comment (ezrasilvera) [reply scripts/update-release.sh:3]: @MikeSpreitzer do we need that for RC?  docs are not uploaded for RC and also the readme is not changed (as the RC is not the "latest version" we want to point to

File: scripts/update-release.sh (hunk 2 above)
Comment (ezrasilvera) [reply scripts/update-release.sh:17]: I used what we have now.

File: scripts/update-release.sh (hunk 1 above)
Comment (MikeSpreitzer) [reply scripts/update-release.sh:23]: https://github.com/kubestellar/kubestellar/blob/main/docs/content/direct/release.md#step-by-step makes no exception for test releases. The editing of the README remark about latest release needs some nuanced handling.

Comment (ezrasilvera) [reply scripts/update-release.sh:23]: Done. -rcX is supported as well

Comment (effi-ofer) [reply scripts/update-release.sh:23]: You should update the comment text to reflect this change.  Can be done separately.

**Review by effi-ofer — APPROVED** [review]

**Pull request merged**